packages/stark-core/src/modules/http/builder/http-request-builder.ts

```diff
@@ -1,3 +1,4 @@
+/* tslint:disable completed-docs*/
```

Comment:
The correct format is: `/* tslint:disable:completed-docs */` otherwise all rules are disabled ;)